CRM/Import/Parser.php

```diff
@@ -293,6 +293,58 @@ public function &getActiveFieldParams() {
   }
 
   /**
```

Comment:
Can you please say what this function does in this comment?

Reply:
Done.